Kitodo/src/test/java/org/kitodo/selenium/LoginST.java

```diff
@@ -1,44 +0,0 @@
-/*
```

Comment:
Is removing this selenium test intended? The login itself should be tested or not?

Reply:
Yes, this test abolition is intentional.

The test checked whether the login was aborted with a warning, if the index was incomplete. This was a bad situation because the application display was based on the index and so incorrect or incomplete data was displayed. This situation will no longer exist in the future, as indexing will only be effective for searching metadata (and possibly speeding up the database search for special columns or queries), but in future, the main application will work without an index.

Whether a login is generally possible is tested in _every_ Selenium test, as the user must log in first.